test/src/org/labkey/test/pages/cds/XAxisVariableSelector.java

```diff
@@ -126,9 +130,88 @@ public void setCellType(String... value)
         super.setAssayDimension(XPATHID, AssayDimensions.CellType, value);
     }
 
-    public void setAxisType(String... value)
+    public void setPlotType(String... value)
     {
-        super.setAssayDimension(XPATHID, AssayDimensions.AxisType, value);
+        super.setAssayDimension(XPATHID, AssayDimensions.PlotType, value);
+    }
+
+    public List<String> getTimePointPlotTypeOptions()
+    {
+        return getTimePointAdvancedOptions("Plot type");
+    }
+
+    public String getTimePointAdvancedOptionSelectedValue(String optionLabel)
+    {
+        _test.mouseOver(Locator.tagWithText("div", optionLabel + ":"));
+        String xpathDimField = getTimeOptionAdvancedOptionXPathDimField(optionLabel);
+        Locator.XPathLocator locDimField = Locator.xpath(xpathDimField);
+        if (_test.isElementPresent(locDimField))
+        {
+            WebElement element = locDimField.findElement(_test.getDriver());
+            if (element != null)
+                return element.getText();
+        }
+        return null;
+    }
+
+    public List<String> getTimePointAdvancedOptions(String optionLabel)
+    {
+        _test.mouseOver(Locator.tagWithText("div", optionLabel + ":"));
+        List<String> options = new ArrayList<>();
+        String xpathDimField = getTimeOptionAdvancedOptionXPathDimField(optionLabel);
+        String xpathDimDropDown = "//div[contains(@class, 'advanced-dropdown')][not(contains(@style, 'display: none'))]";
+
+        Locator.XPathLocator locDimField = Locator.xpath(xpathDimField);
+
+        if (_test.isElementPresent(locDimField))
+        {
+            _test.longWait().until(LabKeyExpectedConditions.animationIsDone(locDimField));
+            _test.click(locDimField);
```

Comment:
As above, use `findElementOrNull` for elements that might not be present. In this case, the test will find the element three times.